content/patterns/radio/examples/radio-activedescendant.html

```diff
@@ -155,53 +147,50 @@ <h2 id="kbd_label">Keyboard Support</h2>
       </section>
 
       <section>
-        <h2 id="rps_label">Role, Property, State, and Tabindex Attributes</h2>
+        <h2 id="rps_label">ロール、プロパティ、ステート、及び tabindex 属性</h2>
         <table aria-labelledby="rps_label" class="data attributes">
           <thead>
             <tr>
-              <th scope="col">Role</th>
-              <th scope="col">Attributes</th>
-              <th scope="col">Element</th>
-              <th scope="col">Usage</th>
+              <th scope="col">ロール</th>
+              <th scope="col">属性</th>
+              <th scope="col">要素</th>
+              <th scope="col">使用法</th>
             </tr>
           </thead>
           <tbody>
             <tr data-test-id="radiogroup-role">
               <th scope="row"><code>radiogroup</code></th>
               <td></td>
               <td><code>ul</code></td>
-              <td>Identifies the element as a container for a group of <code>radio</code> buttons.</td>
+              <td><code>radio</code> ボタンのグループのコンテナとして要素を特定。</td>
             </tr>
             <tr data-test-id="radiogroup-aria-labelledby">
               <td></td>
               <th scope="row"><code>aria-labelledby="[IDREF]"</code></th>
               <td><code>ul</code></td>
-              <td>Refers to the element that contains the label of the radio group.</td>
+              <td>ラジオグループのラベルを含む要素を参照。</td>
```

Comment:
体言止めではなく言い切りの形になおしました

```suggestion
              <td>ラジオグループのラベルを含む要素を参照する。</td>
```